app/tests/spec/models/auth_brokers/fx-desktop.js

```diff
@@ -156,10 +158,34 @@ define([
           })
           .then(function () {
             assert.equal(data.email, 'testuser@testuser.com');
+            assert.isFalse(data.verified);
             assert.isTrue(data.verifiedCanLinkAccount);
             assert.isTrue(channelMock.send.called);
           });
       });
+
+      it('indicates whether the account is verified', function () {
+        // set account as verified
+        account.set('verified', true);
+
+        var data;
+        sinon.stub(channelMock, 'send', function (message, _data, done) {
+          if (message === 'can_link_account') {
+            return done(null, { data: { ok: true }});
+          } else if (message === 'login') {
+            data = _data;
```

Comment:
Can we do the `assert.isTrue(_data.verified)` here? Seems odd (although I'm guessing a good reason) why it needs to be communicated out through the `data` local.